### DISCUSSION TITLE
Graphical/Accessibility option to disable flickering animations

### DISCUSSION THREAD

**Discussion by dpogue** [H-uru/Plasma#1115] [Ideas & Proposals]
Flickering animations, particularly lights that strobe, can pose accessibility problems for players. Many people find them disorienting or headache inducing, and at worst they can trigger seizures.

There are a handful of areas in Uru with flickering animations that some people find uncomfortable:
* Green light in the Gahreesen prison tunnels
* Red light in Tetsonot pod
* Static background on imagers

My proposal is to add a preference to the game to disable these animations, which can be used by a global Python script to enable/disable the relevant object or layer animations.

There's probably some discussion warranted on whether this should be a per-player preference (stored as a Chronicle node in the Vault) or as a per-install preference (similar to most of the other options, such as subtitles).

**Reference:** Web Content Accessibility Guidelines: https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html

**Comment by Deledrius** [comment]
This is a good idea.  It seems to make the most sense for it to be account-wide, so the closest we can do would be a per-install preference.
